src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -359,7 +359,7 @@ public void addInstructor(final String instructorGroup, final String instructorN
         participation3 = participationRepo.save(participation3);
 
         modelingSubmission1.setParticipation(participation1);
-        textSubmission.setParticipation(participation1);
+        textSubmission.setParticipation(participation2);
```

Comment:
The exercise connected to participation should be the same as the one connected to submission.